packages/admin/admin/src/dateTime/DateTimeRangePicker.tsx

```diff
@@ -0,0 +1,175 @@
+import { Calendar } from "@comet/admin-icons";
+import { type ComponentsOverrides, css, inputLabelClasses, type Theme, useThemeProps } from "@mui/material";
+import { type DateTimeRangePickerProps as MuiDateTimeRangePickerProps } from "@mui/x-date-pickers-pro";
+import { type ComponentType, lazy, type ReactNode, Suspense, useState } from "react";
+
+import { ClearInputAdornment as CometClearInputAdornment } from "../common/ClearInputAdornment";
+import { OpenPickerAdornment } from "../common/OpenPickerAdornment";
+import { ReadOnlyAdornment } from "../common/ReadOnlyAdornment";
+import { createComponentSlot } from "../helpers/createComponentSlot";
+import { type ThemedComponentBaseProps } from "../helpers/ThemedComponentBaseProps";
+import { isValidDate } from "./utils";
+
+export type DateTimeRange = {
+    start: Date | null;
+    end: Date | null;
+};
```

Comment:
date-fns has an Interval type, is this something we could use?

https://date-fns.org/v4.1.0/docs/interval#types/Interval/2367

Reply:
Unfortunately `null` can't be used with `Interval`, it must extend `Date` or be `number | string`